projects/observability/src/shared/components/gauge/gauge.component.ts

```diff
@@ -13,22 +13,16 @@ import { Arc, arc, DefaultArcObject } from 'd3-shape';
             [attr.d]="rendererData.backgroundArc"
             *ngIf="rendererData.radius > ${GaugeComponent.GAUGE_MIN_RADIUS_TO_SHOW_PATH}"
           />
-          <g
-            class="input-data"
-            *ngIf="rendererData.data"
-            htTooltip="{{ rendererData.data.value }} of {{ rendererData.data.maxValue }}"
-          >
-            <path
-              class="value-ring"
-              *ngIf="rendererData.radius > ${GaugeComponent.GAUGE_MIN_RADIUS_TO_SHOW_PATH}"
-              [attr.d]="rendererData.data.valueArc"
-              [attr.fill]="rendererData.data.threshold.color"
-            />
-            <text x="0" y="-4" class="value-display" [attr.fill]="rendererData.data.threshold.color">
-              {{ rendererData.data.value }}
-            </text>
-            <text x="0" y="20" class="label-display">{{ rendererData.data.threshold.label }}</text>
-          </g>
+          <ng-template *ngIf="rendererData.data">
```

Comment:
`ng-template` wouldn't be appropriate here (a structural directive like *ngIf already makes a template) - maybe this was meant to be `ng-container` ?

Reply:
Removed it